data/json/items/armor/ballistic_armor.json

```diff
@@ -927,43 +927,32 @@
-    "weight": "2860 g",
+    "//": "Based on a true story related to the real-world design",
+    "description": "This type of ballistic vest once promised to be the future of body armor, claiming to allow a greater range of movement than conventional ballistic vests as well as better protection.  Created by the now-defunct company Meridian Ballistics, it features a unique design consisting of overlapping ceramic disks, and was once famously demonstrated on TV absorbing high-energy rifle rounds at point blank.\n\nThe US Army quickly snapped up several hundred units of the armor, but testing soon revealed that they were extremely heavy, weighing almost fifty pounds, and that the design failed spectacularly in environmental tests. After a series of investigations, claims of fraud, and court hearings, Meridian went bankrupt, and the Department of Justice subsequently dropped this design from its certifications.\n\nThere were no known users of these vests by the time that the world ended; this one was either an original production in the hands of a collector, or a similar design from a different company.  Whatever the case may be, you have one now, in all its unwieldy glory.",
+    "weight": "21545 g",
     "volume": "6 L",
-    "price": 190000,
-    "price_postapoc": 6000,
-    "to_hit": -3,
+    "price": "5000 USD",
+    "price_postapoc": "60 USD",
+    "to_hit": { "grip": "bad", "length": "short", "surface": "any", "balance": "uneven" },
     "bashing": 6,
-    "material": [ "lightceramic", "kevlar_layered" ],
+    "material": [ "ceramic", "kevlar_layered" ],
     "symbol": "[",
     "looks_like": "kevlar",
-    "color": "light_red",
+    "color": "yellow",
     "warmth": 12,
-    "material_thickness": 3,
-    "//": "Ceramic disks inside block addition of steel plating modification",
     "flags": [ "STURDY", "OUTER", "NO_REPAIR" ],
-    "armor": [ { "encumbrance": 8, "coverage": 95, "covers": [ "torso" ] } ]
-  },
-  {
-    "id": "dragonskinempty",
-    "repairs_like": "survivor_suit",
-    "type": "ARMOR",
-    "category": "armor",
-    "name": { "str": "lamellar Kevlar vest" },
-    "description": "A lamellar Kevlar vest, there are intricately cut voids usually filled with stitched in ceramic disks, you could repair the stitching if needed",
-    "weight": "2060 g",
-    "volume": "4 L",
-    "price": 1900,
-    "price_postapoc": 2000,
-    "to_hit": -3,
-    "bashing": 6,
-    "material": [ "kevlar_layered" ],
-    "symbol": "[",
-    "looks_like": "dragonskin",
-    "color": "light_red",
-    "warmth": 10,
-    "material_thickness": 2,
-    "flags": [ "STURDY", "OUTER" ],
-    "armor": [ { "encumbrance": 6, "coverage": 25, "covers": [ "torso" ] } ]
+    "//2": "Ceramic disks in real-life dragon skin are 6.4mm thick; the actual fabric itself wasn't intended to provide high-grade protection, and so it's quite thin here.",
```

Comment:
While lying in bed I can't find any details on the exact makeup of the vest (other than the disks) usually layered kevlar is added on the inside of a vest sort of like the lining of a jacket and goes behind hard armor to stop fragments. The exterior of the vest would probably be kevlar (non layered) or nylon which are iirc .224mm thick and .5mm per sheet. It may *also* have a layered kevlar panel I'm just not sure! So either 1mm of nylon or something like .448mm of kevlar (front + back of vest) should be added no matter what.